mlir/test/Conversion/VectorToLLVM/vector-to-llvm-interface.mlir

```diff
@@ -1785,6 +1799,21 @@ func.func @store_0d(%memref : memref<200x100xf32>, %i : index, %j : index) {
 
 // -----
 
+func.func @store_alignment(%memref : memref<200x100xf32>, %i : index, %j : index) {
+  %val = arith.constant dense<11.0> : vector<4xf32>
```

Comment:
Could this and other constants be function arguments? This way you will reduce the number of Ops in the tests and  align it a bit better with our Testing Guide (https://mlir.llvm.org/getting_started/TestingGuide/#filecheck-best-practices):
> Tests should be minimal, and only check what is absolutely necessary.